packages/core/server/src/routes/web/api/readme.ts

```diff
@@ -0,0 +1,80 @@
+import buildDebug from 'debug';
+import { FastifyInstance } from 'fastify';
+import _ from 'lodash';
+
+import sanitizyReadme from '@verdaccio/readme';
+
+const debug = buildDebug('verdaccio:api:whoami');
+export const NOT_README_FOUND = 'ERROR: No README data found!';
+
+function getReadme(fastify: FastifyInstance, request: any, packageName, callback) {
+  fastify.storage.getPackage({
+    name: packageName,
+    uplinksLook: true,
+    req: request.raw,
+    callback: function (err, readme): void {
+      debug('readme pkg %o', readme?.name);
+      if (err) {
+        callback(err);
+        return;
+      }
+      try {
+        let parsedReadme;
+        try {
+          parsedReadme = parseReadme(readme.name, readme.readme);
+        } catch {
+          parsedReadme = sanitizyReadme(NOT_README_FOUND);
+        }
```

Comment:
No need the try/catch, the `parseReadme` already does it. Remove the try catch clause and use only
```
const parsedReadme = parseReadme(readme.name, readme.readme);
```